packages/headless/src/api/search/search-api-params.ts

```diff
@@ -133,6 +134,16 @@ export interface AutomaticFacetsParams {
   };
 }
 
+export interface PipelineRuleParams {
```

Comment:
Yeah i figured it might change as new pipeline rule parameters were added. Given I had no use for any of the intermediate interfaces I opted for what i thought was more economical. It seems pretty inconsequential now but I like your interfaces and they're already typed out so i'll use them 😉